src/govuk/components/table/_index.scss

```diff
@@ -47,4 +47,25 @@
     display: table-caption;
     text-align: left;
   }
+
+  // Modifiers that make captions look more like their equivalent headings
+
+  .govuk-table__caption--xl {
+    @include govuk-font($size: 48, $weight: bold);
+    margin-bottom: govuk-spacing(3);
+  }
+
+  .govuk-table__caption--l {
+    @include govuk-font($size: 36, $weight: bold);
+    margin-bottom: govuk-spacing(3);
+  }
+
+  .govuk-table__caption--m {
+    @include govuk-font($size: 24, $weight: bold);
+    margin-bottom: govuk-spacing(3);
```

Comment:
In the [typography section of the Design System](https://design-system.service.gov.uk/styles/typography/#headings), the margin-bottom for --m and --s is slightly smaller than the --l heading. Does it look odd to follow these margin styles on the table component?

Reply:
Good question. I followed the code from the [fieldset legend modifiers](https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/components/fieldset/_index.scss#L37-L54), which only add a margin-bottom of `3` for the `xl`, `l` and `m` modifiers.

Reply:
Thanks so much for clarifying @frankieroberto I'm happy to approve :)